tests/plan/import/basic.sh

```diff
@@ -95,13 +116,29 @@ rlJournalStart
         rlAssertGrep "enabled true" $rlRun_LOG
     rlPhaseEnd
 
+    rlPhaseStartTest "Make sure local plan can disable imported plan"
+        rlRun -s "tmt run discover -v plan -n '/plans/disabled'" 2 "Expect error"
```

Comment:
```suggestion
        rlRun -s "tmt run discover -v --remove plan -n '/plans/disabled'" 2 "Expect error"
```

Let's remove runs when done to prevent too many leftovers on the system. Please also add for other `tmt run` commands.

Reply:
Added in ededde81b7a5d548416cc8eff19d1b956773d91f.